pkg/daemon/ceph/osd/volume.go

```diff
@@ -802,7 +802,7 @@ func GetCephVolumeRawOSDs(context *clusterd.Context, clusterInfo *client.Cluster
 	return osds, nil
 }
 
-func callCephVolume(context *clusterd.Context, args ...string) (string, error) {
+func callCephVolume(context *clusterd.Context, action string, args ...string) (string, error) {
```

Comment:
could this be a bool?
```suggestion
func callCephVolume(context *clusterd.Context, requiresCombinedOutput bool, args ...string) (string, error) {
```